pallets/funding/src/types.rs

```diff
@@ -284,12 +284,12 @@ pub mod storage_types {
 		pub status: ProjectStatus,
 		/// When the different project phases start and end
 		pub phase_transition_points: PhaseTransitionPoints<BlockNumber>,
-		/// Fundraising target amount in USD equivalent
-		pub fundraising_target: Balance,
+		/// Fundraising target amount in USD (6 decimals)
+		pub fundraising_target_usd: Balance,
```

Comment:
@felixpolimec: Rename from `fundraising_target` to `fundraising_target_usd` in `ProjectDetails`.